common/src/main/java/org/cloudfoundry/autosleep/config/Config.java

```diff
@@ -113,6 +113,12 @@
         String SECRET = "secret";
 
     }
+    
+    interface OrgEnrollmentParameters {
+        enum EnrolledState {
+            backoffice_enrolled, backoffice_opted_out
+        }
+    }
```

Comment:
Ok. I'll introduce an EnrollmentConfig class into the same package. Let me know if you prefer any other name.